Build failure:
```
======================================================================
FAIL: corehq.apps.es.tests.test_sms:TestSMSES.test_processed_or_incoming
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/es/tests/test_sms.py", line 35, in test_processed_or_incoming
    self.checkQuery(query, json_output)
  File "/mnt/commcare-hq-ro/corehq/apps/es/tests/utils.py", line 16, in checkQuery
    self.assertEqual(raw_query, json_output, msg=msg)
AssertionError: {'que[71 chars]}, {'or': ({'not': {'term': {'direction': 'o'}[109 chars]0000} != {'que[71 chars]}, {'not': {'and': ({'term': {'direction': 'o'[101 chars]0000}
Diff is 947 characters long. Set self.maxDiff to None to see it. : Expected Query:
{
    "query": {
        "filtered": {
            "filter": {
                "and": [
                    {
                        "term": {
                            "domain.exact": "demo"
                        }
                    },
                    {
                        "not": {
                            "and": [
                                {
                                    "term": {
                                        "direction": "o"
                                    }
                                },
                                {
                                    "term": {
                                        "processed": false
                                    }
                                }
                            ]
                        }
                    },
                    {
                        "match_all": {}
                    }
                ]
            },
            "query": {
                "match_all": {}
            }
        }
    },
    "size": 1000000
}
Generated Query:
{
    "query": {
        "filtered": {
            "filter": {
                "and": [
                    {
                        "term": {
                            "domain.exact": "demo"
                        }
                    },
                    {
                        "or": [
                            {
                                "not": {
                                    "term": {
                                        "direction": "o"
                                    }
                                }
                            },
                            {
                                "not": {
                                    "term": {
                                        "processed": false
                                    }
                                }
                            }
                        ]
                    },
                    {
                        "match_all": {}
                    }
                ]
            },
            "query": {
                "match_all": {}
            }
        }
    },
    "size": 1000000
}
```

--- a/corehq/apps/es/tests/test_sms.py
+++ b/corehq/apps/es/tests/test_sms.py
@@ -14,12 +14,14 @@ class TestSMSES(ElasticTestMixin, SimpleTestCase):
                         "and": [
                             {"term": {"domain.exact": "demo"}},
                             {
-                                "not": {
-                                    "and": (
-                                        {"term": {"direction": "o"}},
-                                        {"term": {"processed": False}},
-                                    )
-                                }
+                                "or": (
+                                    {
+                                        "not": {"term": {"direction": "o"}},
+                                    },
+                                    {
+                                        "not": {"term": {"processed": False}},
+                                    }
+                                ),
                             },
                             {"match_all": {}},
                         ]

